src/main/java/duke/DukeDeadlineIncorrectException.java

```diff
@@ -0,0 +1,7 @@
+package duke;
+
+public class DukeDeadlineIncorrectException extends DukeException {
```

Comment:
I like that you specifically crafted exceptions for your program. I was thinking if these should be packaged into `exceptions` as they are all related.

Let me know!